<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="ru.stqa.pft.addressbook.tests.NewContactCreationTests" tests="3" skipped="0" failures="0" errors="0" timestamp="2020-03-18T18:38:19" hostname="WL357543" time="13.799">
  <properties/>
  <testcase name="testNewContactCreation[0](NewContactData{id='0', lastname='lastname 0', name='name 0'})" classname="ru.stqa.pft.addressbook.tests.NewContactCreationTests" time="5.281"/>
  <testcase name="testNewContactCreation[1](NewContactData{id='0', lastname='lastname 1', name='name 1'})" classname="ru.stqa.pft.addressbook.tests.NewContactCreationTests" time="4.228"/>
  <testcase name="testNewContactCreation[2](NewContactData{id='0', lastname='lastname 2', name='name 2'})" classname="ru.stqa.pft.addressbook.tests.NewContactCreationTests" time="4.287"/>
  <system-out><![CDATA[]]></system-out>
  <system-err><![CDATA[Security framework of XStream not initialized, XStream is probably vulnerable.
]]></system-err>
</testsuite>
